File: backend/services/stockservice/src/main/java/com/api/stockservice/domain/event/PoductEvents/UpdateProductEvent.java
package com.api.stockservice.domain.event.PoductEvents;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UpdateProductEvent {
    public String Id ;
    public String MarqueName;
    public String MarqueIcon;
    public String Name;
    public String Description;
    public Double Price;
    public Integer Quantity;
    public String Thumbnail;
    public String CategoryId;
}

File: backend/services/MarketplaceService/src/MarketplaceService.Domain/Events/ProductEvents/UpdateProductEvent.cs
﻿using EventsContracts.EventsContracts;

namespace MarketplaceService.Domain.Events.ProductEvents
{
    public class UpdateProductEvent : IupdateProductEvent
    {
        public string Id { get; set; }
        public string MarqueName { get; set; }
        public string MarqueIcon { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public double Price { get; set; }
        public int Quantity { get; set; }
        public string Thumbnail { get; set; }
        public string CategoryId { get; set; }
    }
}
